features/onbekend_waardes.feature

```diff
@@ -3,87 +3,298 @@
+  Het gaat om de volgende properties en standaardwaardes van de persoon:
+  | property                                         | standaardwaarde  |
+  | ------------------------------------------------ | ---------------- |
+  | naam.geslachtsnaam                               | .                |
+  | verblijfplaats.straat                            | .                |
+  | verblijfplaats.huisnummer                        | 0                |
+  | verblijfplaats.woonplaats                        | .                |
+  | verblijfplaats.nummeraanduidingIdentificatie     | 0000000000000000 |
+  | verblijfplaats.adresseerbaarObjectIdentificatie  | 0000000000000000 |
+  | ouders.naam.geslachtsnaam                        | .                |
+  | partners.naam.geslachtsnaam                      | .                |
+  | partners.soortVerbintenis                        | .                |
+  | kinderen.naam.geslachtsnaam                      | .                |
+  | reisdocumentnummers                              | .........        |
+
+  Abstract Scenario: onbekend waarde "<waarde>" voor <veld>
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    En de persoon heeft de volgende <groep> gegevens
+    | naam   | waarde   |
+    | <veld> | <waarde> |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                             |
+    | type                | RaadpleegMetBurgerservicenummer    |
+    | burgerservicenummer | 999992934                          |
+    | fields              | burgerservicenummer,<groep>.<veld> |
+    Dan bevat de persoon met burgerservicenummer '999992934' alleen de volgende gegevens
+    | naam                  | waarde    |
+    | burgerservicenummer   | 999992934 |
+
+    Voorbeelden:
+    | groep          | veld                              | waarde           |
+    | naam           | geslachtsnaam                     | .                |
+    | verblijfplaats | straat                            | .                |
+    | verblijfplaats | huisnummer                        | 0                |
+    | verblijfplaats | woonplaats                        | .                |
+    | verblijfplaats | nummeraanduidingIdentificatie     | 0000000000000000 |
+    | verblijfplaats | adresseerbaarObjectIdentificatie  | 0000000000000000 |
+
+  Scenario: onbekend waarde in een partner voor soortVerbintenis
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    En de persoon heeft een partner met de volgende gegevens
+    | naam                        | waarde             |
+    | burgerservicenummer         | 999992935          |
+    En de <relatie> met burgerservicenummer '999992935' heeft de volgende soortVerbintenis gegevens
```

Comment:
Hier is een <relatie> placeholder gedefinieerd, maar de Scenario is niet Abstract. Wordt het volgende bedoeld?

```
      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
      | naam                      | waarde   |
      | soort verbintenis (15.10) | .        |
```

Reply:
ik heb nu toch opgenomen:
```
En de partner met burgerservicenummer '999992935' heeft de volgende soortVerbintenis gegevens
    | naam        | waarde   |
    | code        | .        |
```

of wil je dit in de automatisering oplossen? soortVerbintenis is een object van class Waardetabel. Weet je code dan zelf dat deze waarde uit code moet worden gelezen?

Of zou het moeten zijn:
```
| naam                  | waarde    | 
| burgerservicenummer   | 999992935 | 
| soortVerbintenis.code | .         |
```

Hoe wil je het hebben?

Reply:
Graag mijn voorstel. In de automation code wordt een lijst bijgehouden om GBA termen te mappen naar properties. 'soort verbintenis (15.10)' wordt gemapt naar 'soortVerbintenis.code' property. Hierdoor is het mogelijk/makkelijker om in de toekomst alleen de automation code aan te passen/wijzigen (en niet de scenario's/feature files) om te kunnen persisten naar een kolom in een tabel van een database

Reply:
duidelijk. done.